src/templates/vibso_classes.tsv

```diff
@@ -15,8 +15,8 @@ obo:VIBSO_0000011	diffraction grating		https://github.com/NFDI4Chem/VibrationalS
 obo:VIBSO_0000013	excitation wavelength setting		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/48	A setting datum that specifies the wavelength of an electromagnetic radiation source (e.g. a laser) which has the function to excite the energy state of a target material.		owl:Class	setting datum	ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devices.			
 obo:VIBSO_0000014	objective magnification factor setting	magnification|magnification setting	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/24 	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519|ORCID:0000-0002-2239-3955	100x					
-obo:VIBSO_0000015	spectral window range maximum	acquisition band maximum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A data item that is the part of a Raman spectrum which represents the maximum wavenumber included in the spectrum and which is calculated from the windows size and window center.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	210 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.			
-obo:VIBSO_0000016	spectral window range minimum	acquisition band minimum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A data item that is the part of a Raman spectrum which represents the minimum wavenumber included in the spectrum and which is calculated from the windows size and window center.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	1960 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.			
+obo:VIBSO_0000015	spectral window range maximum	acquisition band maximum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum which determines the maximum wavenumber included in the spectrum.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	210 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.			
```

Comment:
So far we always used the as wording for the Genus "A setting datum that specifies ...". And how about we define it as not being dependent on being quantized as a wavenumber? My proposal to rewrite to: 
A setting datum that specifies the upper boundary of the spectral window range utilized during the acquisition of spectroscopic data.